### PR TITLE
add bayreuth

### LINKED ISSUE #81: need Beyreuth, Germany

### PR DESCRIPTION
**Pull request by CJ-Wright**
Closes #81

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -420,4 +420,9 @@ uduisburgessen:
   state: North Rhine-Westphalia
   street: Universitaetsstrasse 7
   zip: 45117
-
+bayreuth:
+  aka: [Bayreuth, bayreuth]
+  city: Bayreuth
+  country: Germany
+  departments: {chemistry: Biology, Chemistry & Earth Sciences}
```
Comment (sbillinge) [inline db/institutions.yml]: should be a dict of dicts now

File: db/institutions.yml [hunk 2]
```diff
@@ -420,4 +420,9 @@ uduisburgessen:
...
+  name: Universitat Rayreuth
```
Comment (sbillinge) [inline db/institutions.yml]: typo?

Comment (sbillinge) [reply db/institutions.yml]: also, if we are going with the German naming I think we should make sure to get the accents right.  Please double-check.

Comment (CJ-Wright) [reply db/institutions.yml]: Yea there needs to be a double dot over the A. (I guess I need to look up how to put unicode into things)

Comment (sbillinge) [reply db/institutions.yml]: copy-paste off the website seems to work for the other guys!   It means we need utfenc package (or sthg like that, I have a note somewhere)......

Note also the "Rayreuth"

Comment (CJ-Wright) [reply db/institutions.yml]: Yes I saw the "R". I'll get them both.

**Comment by CJ-Wright** [comment]
Done

**Comment by sbillinge** [comment]
thanks @CJ-Wright !

**Comment by sbillinge** [comment]
do you have any idea why my british_drug_cos branch is failing on
Travis....?  I am foxed.

**Comment by CJ-Wright** [comment]
No, I'll take a look (I need to change how the validation works anyway so it works against the local copy and not the git copy)

**Pull request merged**